neural_search/params/hybrid_search.json

```diff
@@ -0,0 +1,11 @@
+{
+  "index_name": "quora-dense-embedding",
+  "index_body": "indices/quora.json",
+  "corpora_name": "quora",
+  "default_ingest_pipeline": "nlp-default-ingest-pipeline-dense-embedding",
+  "variable_queries": 10000,
+  "warmup_time_period": 5,
+  "index_knn": true,
+  "k": 10,
+  "ingest_percentage": 10
```

Comment:
we need to ingest all corpus, please change percentage to 100. Same for other param files